.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,23 +1,30 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-title: '[FEATURE] <title>'
-labels: 'feature-request'
-assignees: ''
-
 ---
 
-#### Is your feature request related to a problem? Please describe
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
```

Comment:
pinched from Terraform AWS provider - just a more explicit statement of what we want users to do in terms of crowdsourcing feature request priority